pom.xml

```diff
@@ -65,7 +65,7 @@
         <saxon.version>9.9.1-8</saxon.version>
         <log4j.version>2.19.0</log4j.version>
         <junit.version>5.9.2</junit.version>
-        <elasticsearch.version>7.17.7</elasticsearch.version>
+        <elasticsearch.version>7.17.8</elasticsearch.version>
```

Comment:
Why we are not switching to latest release aka 7.17.10 (at 05/05/2023) of elastic search?

Reply:
I just used and tested the first newer version that was marked as fixing the mentioned vulnerability to minimize the potential for incompatibilities, but I guess if ElasticSearch adheres to the semantic versioning guidelines, 7.17.10 should work, too.

Reply:
Just tested it and version 7.17.10 does indeed work as well without problems, so I changed the version in the pull request as you suggested.